Scenes/HUD/ElementTowersMenu.tscn

```diff
@@ -21,14 +21,6 @@ bg_color = Color(0, 0, 0, 1)
 atlas = ExtResource("4_hw31b")
 region = Rect2(0, 256, 128, 128)
 
-[sub_resource type="InputEventKey" id="InputEventKey_gugyc"]
```

Comment:
You have undone the recent addition of shortcuts. Restore it.

Reply:
Done